.gitignore

```diff
@@ -174,7 +174,8 @@ ci_artifacts/*
 build_info/*
 build_outputs/*
 # Local rattler-build clone
-.rb-clone/*
+.rb-clone
```

Comment:
I think we can keep just .rb-clone/**, wdyt?

Reply:
That does not seem to work, git still wants to add just the folder somehow.